ci/environment-2.7.yml

```diff
@@ -29,6 +29,7 @@ dependencies:
   - sphinx_rtd_theme
   - sphinx-gallery
   - tensorflow
+  - testpath<0.4
```

Comment:
What prompted this?

Reply:
I came across https://stackoverflow.com/questions/52582563/pip-install-attributeerror-distinfodistribution-dep-map and https://github.com/conda-forge/testpath-feedstock/issues/7 after a little bit of Google-ing

Reply:
Definitely just a quick fix until a new release is out